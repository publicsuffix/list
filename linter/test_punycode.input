// test:
// - label is punycode
// - label has double minus

// ===BEGIN ICANN DOMAINS===

a.xn--0zwm56d
a.ex--ample.com

// ===END ICANN DOMAINS===
