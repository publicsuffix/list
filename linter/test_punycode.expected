7: error: Punycode found: 'a.xn--0zwm56d'
8: error: Double minus found: 'a.ex--ample.com'
10: warning: No PRIVATE section found
